membership/README.md

```diff
@@ -0,0 +1,120 @@
+budget of the member institution.
+
+CAS and Bedework Supporting Subscription dues are a fixed price regardless of
+member institution budget.
+
+uPortal Supporting Subscription is offered at several levels not tied to
+member institution budget.
+
+### United States (USD)
+
+| Annual Budget USA (USD) | Core Foundation | Sakai Supporting Subscription |
+| ----------------------- | --------------- | ----------------------------- |
+| less than $25 million   |  2,160          |   900                         |
+| less than $100 million  |  4,680          | 1,950                         |
+| less than $750 million  |  9,720          | 4,050                         |
+| less than $4 billion    | 12,600          | 5,250                         |
+| more than $4 billion    | 16,200          | 6,750                         |
+
+**Bedework supporting subscription:** $500
+
+**CAS supporting subscription:** $500
+
+| uPortal Supporting Subscription level | Dues in US dollars |
+| ------------------------------------- | ------------------ |
+| Bronze - uPortal Friend               |  1,000             |
+| Silver - uPortal Booster              |  5,000             |
+| Gold - uPortal Sustainer              | 10,000             |
+| Platinum - uPortal Champion           | 20,000             |
+
+### Canada (CAD)
+
+| Annual Budget (CAD) | Core Foundation | Sakai Supporting Subscription |
+| ----------------------- | --------------- | ----------------------------- |
+| less than $25 million   |  2,851          | 1,188                         |
+| less than $100 million  |  6,177          | 2,574                         |
+| less than $750 million  | 12,830          | 5,346                         |
+| less than $4 billion    | 16,632          | 6,930                         |
+| more than $4 billion    | 21,384          | 8,910                         |
+
+**Bedework supporting subscription:** $660
+
+**CAS supporting subscription:** $660
+
+| uPortal Supporting Subscription level | Dues in Canadian dollars |
+| ------------------------------------- | ------------------------ |
+| Bronze - uPortal Friend               |  1,320                   |
+| Silver - uPortal Booster              |  6,600                   |
+| Gold - uPortal Sustainer              | 13,200                   |
+| Platinum - uPortal Champion           | 26,400                   |
+
+### Eurozone (Euro)
+
+| Annual Budget (Euro)    | Core Foundation | Sakai Supporting Subscription |
+| ----------------------- | --------------- | ----------------------------- |
+| less than $25 million   |  2,052          |   855                         |
+| less than $100 million  |  4,446          | 1,852.50                      |
+| less than $750 million  |  9,234          | 3,847.50                      |
+| less than $4 billion    | 11,970          | 4,987                         |
+| more than $4 billion    | 15,390          | 6,412.50                      |
```

Comment:
It's surreal that dues for a $4B+ corporation are at a half-Euro precision. Maybe round this up to 6,413 under the hypothesis that once your budget is $4B a half Euro is a rounding error?